contrib/mypy/src/python/pants/contrib/mypy/tasks/mypy_task.py

```diff
@@ -60,9 +82,30 @@ def is_non_synthetic_python_target(target):
   def is_python_target(target):
     return isinstance(target, PythonTarget)
 
-  def _calculate_python_sources(self, targets):
-    """Generate a set of source files from the given targets."""
-    python_eval_targets = filter(self.is_non_synthetic_python_target, targets)
+  def _is_tagged_target(self, target: Target) -> bool:
+    return self.get_options().whitelist_tag_name in target.tags
+
+  def _is_tagged_non_synthetic_python_target(self, target: Target) -> bool:
+    return (self.is_non_synthetic_python_target(target) and
+            self._is_tagged_target(target))
+
+  def _all_targets_in_context_are_whitelisted(self, targets: List[Target], targets_in_context: List[Target]) -> bool:
```

Comment:
In general, "in context" means something a bit different: it means "everything passed on the command line, transitively".

Since you are pre-filtering both of these lists, would drop the word "context" (or just inline the body of this method).